src/components/views/read/cvForm/Education.tsx

```diff
@@ -25,15 +25,18 @@ interface Props {
   prevPage: () => void;
   nextStep: () => void;
   prevStep: () => void;
-  cvValues: CVInfoObj;
-  onCvValuesChange: (newCvValues: CVInfoObj) => void;
+  educationData: EducationInfoObj[];
+  onCvValuesChange: (
+    subkey: string,
```

Comment:
To abstract away the subkey part, you could call the component with:
```ts
onChange={(value) => onCvValuesChange('educationInfo', value)}
```
And here the prop would be:
```ts
onChange: (value: EducationInfoObj[]) => void;
```
That way this component is not concerned with the implementation above.

Reply:
So in the interface, the definition will be like this right?
```
interface Props {
  nextPage: () => void;
  prevPage: () => void;
  nextStep: () => void;
  prevStep: () => void;
  educationData: EducationInfoObj[];
  onChange: (value: EducationInfoObj[]) => void;
}
```
And when i call it in the child prop, it will be like:
`onChange={(cvValues.educationInfo) => onCvValuesChange('educationInfo', cvValues.educationInfo)}`

Reply:
Yes, but you should write in the child prop:
```ts
onChange={(newValues) => onCvValuesChange('educationInfo', newValues)}
```
`newValues` is a parameter that is passed by the child component.